## User

This is the beginning of an individual meeting with Medicinal Chemist to discuss your research project.

Here is context for this meeting:

[begin context 1]

ExperimentThermalShift[Samples]⟹Protocol

creates a Protocol object for measuring sample fluorescence during heating and cooling (melting curve analysis) to determine shifts in thermal stability of Samples under varying conditions.



ExperimentThermalShift quantifies alterations in molecular structure through changes in fluorescence emission across a temperature gradient. This assay utilizes either extrinsic fluorescent dyes or intrinsic fluorescence to monitor conformational changes in nucleic acids or proteins. Additionally, sample aggregate formation across a temperature gradient can be measured using static light scattering whereby larger molecules scatter more light. The data generated by this experiment can be used to determine the samples’ melting point or aggregation point which is characteristic of the samples’ sequence, length, concentration, and solvent conditions. Resulting shifts in melting and aggregation points compared to a reference are indicative of sample thermal stability under varying conditions.



Experimental Principles

Figure 1.1: Step1: Samples can be diluted, pooled, mixed, incubated, and/or annealed in preparation for thermal shift analysis. Sample dilution occurs prior to sample pooling. When all samples in a pool are diluted, the total number of dilutions of each sample in the pool must match. When one sample in a pool is diluted and its partner is not diluted, the undiluted pooling partner will be pooled with each dilution of its partner at the desired AliquotAmount. Step2: Prepared samples are transferred into instrument compatible containers. For the Uncle multimode spectrophotometer, samples are loaded into a 16 well quartz capillary strip. For the ViiA 7 Real-Time thermocycler, samples are loaded into a 384-well v-bottom optical assay plate. Step3: Prepared samples are loaded into the instrument and equilibrated to the starting temperature of the following thermal ramp. A total of three capillary strips can be loaded into the Uncle at once. Step4: Sample fluorescence (and static light scattering if desired) measurement during thermal ramp. For the Uncle, fluorescence and static light scattering are measured for each capillary independently. For the ViiA 7, fluorescence measurements are collected at each excitation wavelength simultaneously for all wells. Steps 3 & 4 are repeated for each heating and cooling cycle. Step5: Data is assessed using AnalyzeMeltingPoint to determine the melting temperature of samples. Static light scattering data from the Uncle can also be analyzed to determine relative changes in aggregation temperature of samples.



Instrumentation

Uncle

Model[Instrument, MultimodeSpectrophotometer, "id:4pO6dM508MbX"]

Figure 2.1.1: The Uncle is a multimode spectrophotometer with the capacity to interrogate forty eight samples in one run. Samples must be assayed in Uncle compatible capillary strips. Each strip contains 16 quartz capillaries and each capillary requires 9 uL of sample. Sample fluorescence and static light scattering intensity are excited by 266 nm and 473 nm long-life lasers and detected by a full-spectrum spectrometer consisting of a CCD camera pixel array. Excitation laser power can be specified from opaque to full power (0 - 100% respectively). Sample dynamic light scattering can be quantified by a 660 nm laser diode and detected using an avalanche photodiode module. For all measurements, each capillary is interrogated independently. Thermal profiles can be programmed into the instrument to interrogate samples at multiple temperatures. Capillary strip temperature is controlled by a thermoelectric heating and cooling plate and can range from 15°C to 95°C.

ViiA 7

Model[Instrument, Thermocycler, "id:D8KAEvdqzXYk"]

Figure 2.2.1: The ViiA 7 offers real-time monitoring of target sequence amplification during 384-well plate-based PCR procedures. The thermocycling profile is programmable in the temperature range from 4 °C to 100 °C, with a maximum ramp rate of 3.4 °C/sec. The ViiA 7's excitation wavelengths of 470, 520, 550, 580, 640, and 662 nm and emission wavelengths of 520, 558, 586, 623, 682, and 711 nm allow monitoring of the amplification targets via a variety of dyes or reporters.



Experiment Options

General

Instrument

The instrument used to perform the thermal shift experiment.

Default Value: Automatic

Default Calculation: Automatically set to Model[Instrument, Thermocycler, "ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] for nucleic acid samples and Model[Instrument, MultimodeSpectrophotometer, "Uncle"] for protein samples

Pattern Description: An object of type or subtype Model[Instrument, Thermocycler], Object[Instrument, Thermocycler], Model[Instrument, MultimodeSpectrophotometer], or Object[Instrument, MultimodeSpectrophotometer]

Programmatic Pattern: ObjectP[{Model[Instrument, Thermocycler], Object[Instrument, Thermocycler], Model[Instrument, MultimodeSpectrophotometer], Object[Instrument, MultimodeSpectrophotometer]}] | Automatic

NumberOfReplicates

The number of different analysis wells in which each nested index matching sample is measured.

Default Value: Automatic

Default Calculation: Automatically set to 2 if Instrument is set to a Model[Instrument, MultimodeSpectrophotometer] or Object[Instrument, MultimodeSpectrophotometer].

Pattern Description: Greater than or equal to 2 in increments of 1 or Null.

Programmatic Pattern: (GreaterEqualP[2, 1] | Automatic) | Null

AnalyteType

The type of molecule present in the nested index matching sample that is detected during instrument measurement.

Default Value: Automatic

Default Calculation: Automatically set to the type of specified analytes in the sample definition.

Pattern Description: Oligomer or Protein.

Programmatic Pattern: (Oligomer | Protein) | Automatic

Index Matches to: experiment samples

DynamicLightScatteringCapillaryLoading

The loading method for capillaries when the instrument used is Model[Instrument, MultimodeSpectrophotometer, "Uncle"] . When set to Robotic, capillaries are loaded by liquid handler. when set to Manual, capillaries are loaded by a multichannel pipette.

Default Value: Automatic

Default Calculation: If Automatic, set to Manual if the instrument is Model[Instrument, MultimodeSpectrophotometer, "Uncle"] and to Null otherwise.

Pattern Description: Automatic, Robotic, or Manual or Null.

Programmatic Pattern: ((Automatic | Robotic | Manual) | Automatic) | Null

Sample Preparation

ReactionVolume

The total volume of the reaction including the sample, Buffer, DetectionReagent, and PassiveReference. For protocols using Model[Instrument,MultimodeSpectrophotometer], a minimum of 20 Microliter for the ReactionVolume is required.

Default Value: Automatic

Pattern Description: Greater than or equal to 5 microliters and less than or equal to 40 microliters.

Programmatic Pattern: RangeP[5*Microliter, 40*Microliter] | Automatic

PassiveReference

A temperature insensitive fluorophore used to normalize melting curves.

Default Value: Null

Pattern Description: An object of type or subtype Model[Sample] or Object[Sample] or a prepared sample or Null.

Programmatic Pattern: (ObjectP[{Model[Sample], Object[Sample]}] | _String) | Null

PassiveReferenceVolume

The volume of PassiveReference to add to the final reaction.

Default Value: Automatic

Default Calculation: Automatically set to the manufacturer recommended working concentration of the selected passive reference dye or Null if PassiveReference is not specified.

Pattern Description: Greater than 0 microliters or Null.

Programmatic Pattern: (GreaterP[0*Microliter] | Automatic) | Null

DilutionCurve

The collection of dilutions that are performed on sample prior to sample pooling (if desired) and thermal ramping.

Figure 3.1: Use the DilutionCurve option to create a collection of dilutions that will be performed on the sample.

Default Value: Null

Pattern Description: Fixed Dilution Factor or Fixed Dilution Volume or Null.

Programmatic Pattern: (Null | {{GreaterEqualP[0*Microliter], GreaterEqualP[0*Microliter]}..} | {{GreaterEqualP[0*Microliter], RangeP[0, 1]}..}) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

SerialDilutionCurve

The collection of dilutions that are performed on sample prior to sample pooling (if desired) and thermal ramping. For Serial Dilution Volumes, the Transfer Volume is taken out of the sample and added to a second well with the Diluent Volume of the Diluent. It is mixed and then the Transfer Volume is taken out of that well to be added to a third well. This is repeated to make Number Of Dilutions diluted samples. For example, 100mM sample with a TransferVolume of 20 Microliters, a DiluentVolume of 60 Microliters and a NumberofDilutions of 3 is used, it will create a DilutionCurve of 100mM, 25mM, and 6.25mM with each dilution having a volume of 60 Microliters. For Serial Dilution Factors, the sample is diluted by the dilution factor at each transfer step.

Figure 3.2: Use the SerialDilutionCurve option to create a collection of serial dilutions that will be performed on the sample.

Default Value: Null

Pattern Description: Serial Dilution Factor or Serial Dilution Volumes or Null.

Programmatic Pattern: (Null | {GreaterP[0*Microliter], GreaterEqualP[0*Microliter], GreaterP[1, 1]} | {GreaterEqualP[0*Microliter], {RangeP[0, 1], GreaterP[1, 1]} | {RangeP[0, 1]..}}) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

Diluent

The sample that is used to dilute the sample to make a DilutionCurve or SerialDilutionCurve.

Default Value: Automatic

Default Calculation: Automatically set to Model[Sample,"Nuclease-free Water"] for nucleic acid analytes and Model[Sample,StockSolution,"1x PBS from 10X stock, pH 7"] for protein analytes when SerialDilutionCurve or DilutionCurve are specified and Null when neither SerialDilutionCurve or DilutionCurve are specified.

Pattern Description: An object of type or subtype Model[Sample] or Object[Sample] or a prepared sample or Null.

Programmatic Pattern: ((Null | (ObjectP[{Model[Sample], Object[Sample]}] | _String)) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

DilutionContainer

The containers in which each sample is diluted with the Diluent to make the concentration series, with indices indicating specific grouping of samples if desired.

Default Value: Automatic

Default Calculation: Automatically set as {1,Model[Container,Plate,"96-well 2mL Deep Well Plate"]}.

Pattern Description: {Index, Container} or Null.

Programmatic Pattern: ((Null | {GreaterEqualP[1, 1], ObjectP[{Model[Container], Object[Container]}] | _String}) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

DilutionStorageCondition

The conditions under which any leftover samples from the DilutionCurve or SerialDilutionCurve should be stored after the samples are transferred to the measurement plate.

Default Value: Automatic

Default Calculation: Automatically set to Disposal.

Pattern Description: {AmbientStorage, Refrigerator, Freezer, DeepFreezer, CryogenicStorage, YeastIncubation, YeastShakingIncubation, BacterialIncubation, BacterialShakingIncubation, MammalianIncubation, ViralIncubation, CrystalIncubation, AcceleratedTesting, IntermediateTesting, LongTermTesting, UVVisLightTesting} or Disposal or Null.

Programmatic Pattern: ((Alternatives[SampleStorageTypeP | Disposal]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

DilutionMixVolume

The volume that is pipetted out of and in to the dilution to mix the sample with the Diluent to make the DilutionCurve or SerialDilutionCurve.

Default Value: Automatic

Default Calculation: Automatically set to the smallest dilution volume or half the largest dilution volume, whichever one is smaller.

Pattern Description: Greater than or equal to 0 microliters or Null.

Programmatic Pattern: ((Null | GreaterEqualP[0*Microliter]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

DilutionNumberOfMixes

The number of pipette out and in cycles that is used to mix the sample with the Diluent to make the DilutionCurve or SerialDilutionCurve.

Default Value: Automatic

Default Calculation: Automatically set to 5 when DilutionCurve or SerialDilutionCurve is specified.

Pattern Description: Greater than or equal to 0 and less than or equal to 20 in increments of 1 or Null.

Programmatic Pattern: ((Null | RangeP[0, 20, 1]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

DilutionMixRate

The speed at which the DilutionMixVolume is pipetted out and in of the dilution to mix the sample with the Diluent to make the DilutionCurve or SerialDilutionCurve.

Default Value: Automatic

Default Calculation: Automatically set to 100 Microliter/Second when DilutionCurve or SerialDilutionCurve is specified.

Pattern Description: Greater than or equal to 0.4 microliters per second and less than or equal to 250 microliters per second or Null.

Programmatic Pattern: ((Null | RangeP[0.4*(Microliter/Second), 250*(Microliter/Second)]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

NestedIndexMatchingMix

Indicates if mixing of the nested index matching samples occur inside the destination plate.

Default Value: Automatic

Default Calculation: Automatically set based on whether pooling of the source samples is performed.

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

Index Matches to: experiment samples

NestedIndexMatchingMixType

Indicates the style of motion used to mix the nested index matching samples inside the AliquotContainer.

Default Value: Automatic

Default Calculation: Automatically set to Pipette when NestedIndexMatchingMix is True.

Pattern Description: Pipette or Invert or Null.

Programmatic Pattern: ((Pipette | Invert) | Automatic) | Null

Index Matches to: experiment samples

NestedIndexMatchingNumberOfMixes

The number of times each nested index matching sample is mixed by pipetting or inversion.

Default Value: Automatic

Default Calculation: Automatically set to 5.

Pattern Description: Greater than or equal to 1 and less than or equal to 50 in increments of 1 or Null.

Programmatic Pattern: (RangeP[1, 50, 1] | Automatic) | Null

Index Matches to: experiment samples

NestedIndexMatchingMixVolume

The volume of each nested index matching sample is pipetted up and down to mix.

Default Value: Automatic

Default Calculation: Automatically set to half the total nested index matching volume.

Pattern Description: Greater than or equal to 1 microliter and less than or equal to 50 milliliters or Null.

Programmatic Pattern: (RangeP[1*Microliter, 50*Milliliter] | Automatic) | Null

Index Matches to: experiment samples

NestedIndexMatchingIncubate

Indicates if thermal incubation of the nested index matching samples occur prior to measurement.

Default Value: Automatic

Default Calculation: Automatically set to False if NestedIndexMatchingIncubationTime, NestedIndexMatchingIncubationTemperature, and NestedIndexMatchingAnnealingTime are not specified. Automatically set to True if either NestedIndexMatchingIncubationTime, NestedIndexMatchingIncubationTemperature, or NestedIndexMatchingAnnealingTime are specified.

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

Index Matches to: experiment samples

PooledIncubationTime

Duration for which the nested index matching samples are thermally incubated prior to measurement.

Default Value: Automatic

Default Calculation: Automatically set to Null if NestedIndexMatchingIncubate is False and 5 minutes if NestedIndexMatchingIncubate is True.

Pattern Description: Greater than or equal to 1 minute and less than or equal to 72 hours or Null.

Programmatic Pattern: (RangeP[1*Minute, $MaxExperimentTime] | Automatic) | Null

Index Matches to: experiment samples

NestedIndexMatchingIncubationTemperature

Temperature at which the nested index matching samples are thermally incubated prior to measurement.

Default Value: Automatic

Default Calculation: Automatically set to Null if NestedIndexMatchingIncubate is False and 85 Celsius if NestedIndexMatchingIncubate is True.

Pattern Description: Ambient or greater than or equal to 22 degrees Celsius and less than or equal to 90 degrees Celsius or Null.

Programmatic Pattern: ((RangeP[22*Celsius, 90*Celsius] | Ambient) | Automatic) | Null

Index Matches to: experiment samples

NestedIndexMatchingAnnealingTime

Duration for which the nested index matching samples remain in the thermal incubation instrument before being removed, allowing the system to settle to room temperature after the NestedIndexMatchingIncubationTime has passed.

Default Value: Automatic

Default Calculation: Automatically set to Null if NestedIndexMatchingIncubate is False and 3 Hour if NestedIndexMatchingIncubate is True.

Pattern Description: Greater than or equal to 0 minutes and less than or equal to 72 hours or Null.

Programmatic Pattern: (RangeP[0*Minute, $MaxExperimentTime] | Automatic) | Null

Index Matches to: experiment samples

SampleVolume

The volume of the sample containing the main analyte used in the experiment.

Default Value: Automatic

Default Calculation: Automatically set to 20uL if the instrument is Model[Instrument,Thermocycler,"ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models and 10uL if the instrument is Model[Instrument,MultimodeSpectrophotometer,"Uncle"] or an object with this model.

Pattern Description: Greater than or equal to 0 microliters and less than or equal to 40 microliters.

Programmatic Pattern: RangeP[0*Microliter, 40*Microliter] | Automatic

Index Matches to: experiment samples

Buffer

A sample used to bring each reaction to its reaction volume once all other components have been added.

Default Value: Automatic

Default Calculation: Automatically set to Model[Sample,"Nuclease-free Water"] for nucleic acid analytes and Model[Sample,StockSolution,"1x PBS from 10X stock, pH 7"] for protein analytes.

Pattern Description: An object of type or subtype Model[Sample] or Object[Sample] or a prepared sample or Null.

Programmatic Pattern: ((ObjectP[{Model[Sample], Object[Sample]}] | _String) | Automatic) | Null

Index Matches to: experiment samples

BufferVolume

The volume, of Buffer added to bring the reaction volume up to ReactionVolume.

Default Value: Automatic

Default Calculation: Automatically set to the volume difference between the sum of the reaction components and the reaction volume.

Pattern Description: Greater than or equal to 0 microliters and less than or equal to 40 microliters or Null.

Programmatic Pattern: (RangeP[0*Microliter, 40*Microliter] | Automatic) | Null

Index Matches to: experiment samples

DetectionReagent

The fluorophore or fluorescent dye used to detect melting of the analyte.

Default Value: Automatic

Default Calculation: Automatically set to Model[Sample, StockSolution, "10X SYBR Gold in filtered 1X TBE Alternative Preparation"] for nucleic acid analytes. Automatically set to Model[Sample,StockSolution,"10X SYPRO Orange"] for protein analytes if the instrument is Model[Instrument,Thermocycler,"ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models and Null if the instrument is Model[Instrument,MultimodeSpectrophotometer,"Uncle"] or an object with this model.

Pattern Description: An object of type or subtype Object[Sample] or Model[Sample] or a prepared sample or Null.

Programmatic Pattern: ((ObjectP[{Object[Sample], Model[Sample]}] | _String) | Automatic) | Null

Index Matches to: experiment samples

DetectionReagentVolume

The volume of the DetectionReagent to add to the reaction.

Default Value: Automatic

Default Calculation: Automatically set to the manufacturer's recommendation for the working concentration of the selected detection reagent. In the case of Model[Sample,StockSolution,"10X SYPRO Orange"], the default working concentration is set to 5X.

Pattern Description: Greater than or equal to 0 microliters and less than or equal to 40 microliters or Null.

Programmatic Pattern: (RangeP[0*Microliter, 40*Microliter] | Automatic) | Null

Index Matches to: experiment samples

Melting Curve

MinTemperature

The low temperature of the heating or cooling cycle.

Default Value: 15 degrees Celsius

Pattern Description: Greater than or equal to 4 degrees Celsius and less than or equal to 100 degrees Celsius.

Programmatic Pattern: RangeP[4*Celsius, 100*Celsius]

MaxTemperature

The high temperature of the heating or cooling cycle.

Default Value: 95 degrees Celsius

Pattern Description: Greater than or equal to 4 degrees Celsius and less than or equal to 100 degrees Celsius.

Programmatic Pattern: RangeP[4*Celsius, 100*Celsius]

TemperatureRampOrder

The order of temperature ramping to be performed in each cycle.

Default Value: {Heating, Cooling}

Pattern Description: {Heating, Cooling} or {Cooling, Heating}

Programmatic Pattern: ThermodynamicCycleP

NumberOfCycles

The number of repeated melting and cooling cycles to be performed.

Default Value: Automatic

Default Calculation: Automatically set to 0.5 cycles. Half cycles represent a monotonically ascending or descending temperature ramp and is determined by the first entry in TemperatureRampOrder. More than 0.5 cycles can only be measured on Model[Instrument,MultimodeSpectrophotometer,"Uncle"].

Pattern Description: 0.5, 1, 1.5, 2, 2.5, or 3.

Programmatic Pattern: (0.5 | 1 | 1.5 | 2 | 2.5 | 3) | Automatic

EquilibrationTime

The time between each melting and cooling cycle during which the samples are held at the starting temperature of the next thermal ramp.

Default Value: 1 minute

Pattern Description: Greater than or equal to 0 seconds and less than or equal to 5999 seconds.

Programmatic Pattern: RangeP[0*Second, 99*Minute + 59*Second]

TemperatureRamping

The type of temperature ramp. Linear temperature ramps increase temperature at a constant rate given by TemperatureRampRate. Step temperature ramps increase the temperature by TemperatureRampStep/TemperatureRampStepTime and holds the temperature constant for TemperatureRampStepHold before measurement.

Default Value: Automatic

Default Calculation: Automatically set to Linear if no stepped temperature ramp options are specified. Automatically set to Step if any stepped temperature ramp options are specified.

Pattern Description: Linear or Step.

Programmatic Pattern: (Linear | Step) | Automatic

TemperatureRampRate

The rate at which the temperature is changed in the course of one heating and/or cooling cycle.

Default Value: Automatic

Default Calculation: If TemperatureRamping is Linear, automatically set to 1 Celsius/Minute. If TemperatureRamping is Step, automatically set to the max ramp rate available on the instrument.

Pattern Description: Greater than or equal to 0.0015 degrees Celsius per second and less than or equal to 3.4 degrees Celsius per second.

Programmatic Pattern: RangeP[0.0015*(Celsius/Second), 3.4*(Celsius/Second)] | Automatic

TemperatureResolution

The amount by which the temperature is changed between each data point and the subsequent data point for a given sample during the melting and/or cooling curves.

Figure 3.3: Linear thermal profiles programmed into the ExperimentThermalShift instruments can be customized using the following options: 1) MinTemperature and MaxTemperature set the lower and upper values of the thermal profile respectively and can range from 4 to 100 Celsius. 2) TemperatureRampRate defines the rate at which the temperature changes with time and can range from 0.0015 to 3.4 Celsius per second. 3) EquilibrationTime sets the interval preceding each heating or cooling ramp during which the samples are held at the starting temperature of the heating or cooling ramp. 4) TemperatureResolution determines the change in temperature between measurement reads. The maximum resolution depends on the total number of samples. This option is only available on the Uncle instrument. 5) TemperatureRampOrder sets the order of the heating and cooling ramps and can either start with a heating ramp or a cooling ramp. 6) NumberOfCycles determines how many times the heating and cooling ramps are repeated during the experiment and can range from 0.5 to 3 cycles. The ViiA 7 thermocycler is limited to 0.5 cycles. Each cycle is considered to be a heating ramp plus a cooling ramp in any order.

Default Value: Automatic

Default Calculation: Automatically set to highest possible resolution if the instrument is Model[Instrument,MultimodeSpectrophotometer,"Uncle"] or an object with this model. Automatically set to Null if the instrument is Model[Instrument,Thermocycler,"ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models because temperature resolution is not tunable on this instrument type.

Pattern Description: Greater than or equal to 0 degrees Celsius and less than or equal to 100 degrees Celsius or Null.

Programmatic Pattern: (RangeP[0*Celsius, 100*Celsius] | Automatic) | Null

NumberOfTemperatureRampSteps

The number of step changes in temperature for a heating or cooling cycle.

Figure 3.4: Step thermal profiles programmed into the ExperimentThermalShift instruments can be customized using the following options: 1) MinTemperature and MaxTemperature set the lower and upper values of the thermal profile respectively and can range from 4 to 100 Celsius. 2) TemperatureRampRate defines the rate at which the temperature changes between each step temperature and can range from 0.0015 to 3.4 Celsius per second. 3) NumberOfTemperatureRampSteps sets the number of temperature steps taken between the MinTemperature and MaxTemperature for each heating or cooling ramp. 4) EquilibrationTime sets the interval preceding each heating or cooling ramp during which the samples are held at the starting temperature of the heating or cooling ramp. 5) StepHoldTime sets the length of time samples are held at each temperature during a stepped temperature ramp prior to measurement. 6) TemperatureRampOrder sets the order of the heating and cooling ramps and can either start with a heating ramp or a cooling ramp. 7) NumberOfCycles determines how many times the heating and cooling ramps are repeated during the experiment and can range from 0.5 to 3 cycles. The ViiA 7 thermocycler is limited to 0.5 cycles. Each cycle is considered to be a heating ramp plus a cooling ramp in any order.

Default Value: Automatic

Default Calculation: If TemperatureRamping is Step, automatically set to the absolute value of the difference in MinTemperature and MaxTemperature rounded to the nearest integer. If TemperatureRamping is Linear, automatically set to Null.

Pattern Description: Greater than or equal to 1 and less than or equal to 100 or Null.

Programmatic Pattern: (RangeP[1, 100] | Automatic) | Null

StepHoldTime

The length of time samples are held at each temperature during a stepped temperature ramp prior to fluorescence or static light scattering measurement.

Default Value: Automatic

Default Calculation: Automatically set to 30 Second when TemperatureRamping is "Step" and Null when TemperatureRamping is "Linear".

Pattern Description: Greater than or equal to 1 second and less than or equal to 13500 seconds or Null.

Programmatic Pattern: (RangeP[1*Second, 13500*Second] | Automatic) | Null

Detection

FluorescenceLaserPower

The fluorescence laser power filter used in the experiment. If OptimizeFluorescenceLaserPower is True, this is the laser power at which the optimization starts.

Default Value: Automatic

Default Calculation: Automatically set to 50 Percent if the instrument is Model[Instrument,MultimodeSpectrophotometer,"Uncle"] or an object with this model and Null if the instrument is Model[Instrument,Thermocycler,"ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models.

Pattern Description: 0 percent, 13 percent, 25 percent, 32 percent, 50 percent, or 100 percent or Null.

Programmatic Pattern: (LaserPowerFilterP | Automatic) | Null

Index Matches to: experiment samples

StaticLightScatteringLaserPower

The static light scattering laser power filter used in the experiment. A laser power of 25% is recommended for most proteins. Higher laser powers may be necessary for smaller particles that do not scatter as much light. If OptimizeStaticLightScatteringLaserPower is True, the laser power used in the experiment will match the optimized fluorescence laser power setting.

Default Value: Automatic

Default Calculation: Automatically set to 25 Percent if the instrument is Model[Instrument,MultimodeSpectrophotometer,"Uncle"] or an object with this model and Null if the instrument is Model[Instrument,Thermocycler,"ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models.

Pattern Description: 0 percent, 13 percent, 25 percent, 32 percent, 50 percent, or 100 percent or Null.

Programmatic Pattern: (LaserPowerFilterP | Automatic) | Null

Index Matches to: experiment samples

OptimizeFluorescenceLaserPower

Indicates if prior to thermal ramping, the fluorescence laser power filter should be adjusted such that the sample's spectra at or between LaserOptimizationEmissionWavelengthRange has a peak intensity that falls within the LaserOptimizationTargetEmissionIntensityRange.

Default Value: False

Pattern Description: True or False.

Programmatic Pattern: BooleanP

Index Matches to: experiment samples

OptimizeStaticLightScatteringLaserPower

Indicates if prior to thermal ramping, the static light scattering laser power filter should be adjusted to match the optimized fluorescence laser power filter.

Default Value: False

Pattern Description: True or False.

Programmatic Pattern: BooleanP

Index Matches to: experiment samples

LaserOptimizationEmissionWavelengthRange

Indicates the wavelength range of the sample's spectra used to evaluate optimal laser setting when OptimizeFluorescenceLaserPower is True.

Default Value: Automatic

Default Calculation: Automatically set to Null if OptimizeFluorescenceLaserPower is False. If OptimizeFluorescenceLaserPower is True, automatically set to {300*Nanometer,450*Nanometer}.

Pattern Description: A span from anything greater than or equal to 250 nanometers and less than or equal to 719 nanometers in increments of 1 nanometer to anything greater than or equal to 251 nanometers and less than or equal to 750 nanometers in increments of 1 nanometer or Null.

Programmatic Pattern: (RangeP[250*Nanometer, 719*Nanometer, 1*Nanometer] ;; RangeP[251*Nanometer, 750*Nanometer, 1*Nanometer] | Automatic) | Null

Index Matches to: experiment samples

LaserOptimizationTargetEmissionIntensityRange

Indicates the optimal signal intensity range, expressed as a percentage of the detector saturation intensity, used to evaluate optimal laser setting when OptimizeFluorescenceLaserPower is True.

Default Value: Automatic

Default Calculation: Automatically set to Null if OptimizeFluorescenceLaserPower is False. If OptimizeFluorescenceLaserPower is True, set to {5 Percent, 20 Percent}.

Pattern Description: A span from anything greater than or equal to 0.1 percent and less than or equal to 99.9 percent in increments of 0.1 percent to anything greater than or equal to 0.2 percent and less than or equal to 100 percent in increments of 0.1 percent or Null.

Programmatic Pattern: (RangeP[0.1*Percent, 99.9*Percent, 0.1*Percent] ;; RangeP[0.2*Percent, 100*Percent, 0.1*Percent] | Automatic) | Null

Index Matches to: experiment samples

ExcitationWavelength

The wavelength of light used to excite the reporter component of the detection reagent.

Default Value: Automatic

Default Calculation: Automatically set to the available excitation wavelength closest to the excitation maximum of the corresponding detection reagent if the instrument is Model[Instrument, Thermocycler, "ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models. Automatically set to 266 nm if the instrument is Model[Instrument, MultimodeSpectrophotometer, "Uncle"] or an object with this model

Pattern Description: Greater than or equal to 266 nanometers and less than or equal to 670 nanometers.

Programmatic Pattern: RangeP[266*Nanometer, 670*Nanometer] | Automatic

Index Matches to: experiment samples

EmissionWavelength

The wavelength of emitted light recorded.

Default Value: Automatic

Default Calculation: Automatically set to the available emission wavelength closest to the emission maximum of the corresponding detection reagent if the instrument is Model[Instrument, Thermocycler, "ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models. Automatically set to Null if the instrument is Model[Instrument, MultimodeSpectrophotometer, "Uncle"] or an object with this model

Pattern Description: Greater than or equal to 250 nanometers and less than or equal to 720 nanometers or Null.

Programmatic Pattern: (RangeP[250*Nanometer, 720*Nanometer] | Automatic) | Null

Index Matches to: experiment samples

MinEmissionWavelength

The minimum wavelength of emitted light recorded for the sample fluorescence spectra.

Default Value: Automatic

Default Calculation: Automatically set to 250nm if the instrument is Model[Instrument, MultimodeSpectrophotometer, "Uncle"] or an object with this model and Null if the instrument is Model[Instrument, Thermocycler, "ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models.

Pattern Description: Greater than or equal to 250 nanometers and less than or equal to 720 nanometers or Null.

Programmatic Pattern: (RangeP[250*Nanometer, 720*Nanometer] | Automatic) | Null

Index Matches to: experiment samples

MaxEmissionWavelength

The maximum wavelength of emitted light recorded for the sample fluorescence spectra.

Default Value: Automatic

Default Calculation: Automatically set to 720nm if the instrument is Model[Instrument, MultimodeSpectrophotometer, "Uncle"] or an object with this model and Null if the instrument is Model[Instrument, Thermocycler, "ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models.

Pattern Description: Greater than or equal to 250 nanometers and less than or equal to 720 nanometers or Null.

Programmatic Pattern: (RangeP[250*Nanometer, 720*Nanometer] | Automatic) | Null

Index Matches to: experiment samples

StaticLightScatteringExcitationWavelength

The wavelength(s) of light used to illuminate static light scattering measurements.

Default Value: Automatic

Default Calculation: Automatically set to {266 Nanometer, 473 Nanometer} if the instrument is Model[Instrument, MultimodeSpectrophotometer, "Uncle"] or an object with this model and Null if the instrument is Model[Instrument, Thermocycler, "ViiA 7"] or Model[Instrument, Thermocycler, "QuantStudio 7 Flex"] or an object with these models.

Pattern Description: A selection of one or more of 266 nanometers or 473 nanometers or Null.

Programmatic Pattern: (DuplicateFreeListableP[266*Nanometer | 473*Nanometer] | Automatic) | Null

Index Matches to: experiment samples

DynamicLightScattering

Indicates if at least one DLS measurement of each sample is made during the experiment.

Default Value: False

Pattern Description: True or False.

Programmatic Pattern: BooleanP

DynamicLightScatteringMeasurements

Describes if a DLS measurement of each sample will be made before thermal ramping, after thermal ramping, or both before and after thermal ramping.

Default Value: Automatic

Default Calculation: Automatically set to Null if DynamicLightScattering is False and to {Before,After} otherwise.

Pattern Description: {Before}, {After}, or {Before, After} or Null.

Programmatic Pattern: (({Before} | {After} | {Before, After}) | Automatic) | Null

DynamicLightScatteringMeasurementTemperatures

Indicates the temperatures at which DLS measurements will be acquired. If only a single measurement is required, provide only one temperature. If DynamicLightScatteringMeasurements->{Before,After}, provide a matched list of temperatures.

Default Value: Automatic

Default Calculation: Automatically set to Null if DynamicLightScattering is False and to the temperature of the adjacent thermal ramp otherwise.

Pattern Description: {Before, After} or greater than or equal to 15 degrees Celsius and less than or equal to 95 degrees Celsius or Null.

Programmatic Pattern: ((RangeP[15*Celsius, 95*Celsius] | {RangeP[15*Celsius, 95*Celsius], RangeP[15*Celsius, 95*Celsius]}) | Automatic) | Null

NumberOfDynamicLightScatteringAcquisitions

For each DLS measurement, the number of series of speckle patterns that are each collected over the AcquisitionTime to create the measurement's autocorrelation curve.

Default Value: Automatic

Default Calculation: Automatically set to Null if DynamicLightScattering is False and to 4 if DynamicLightScattering is True.

Pattern Description: Greater than or equal to 1 and less than or equal to 20 in increments of 1 or Null.

Programmatic Pattern: (RangeP[1, 20, 1] | Automatic) | Null

DynamicLightScatteringAcquisitionTime

For each DLS measurement, the length of time that each acquisition generates speckle patterns to create the measurement's autocorrelation curve.

Default Value: Automatic

Default Calculation: Automatically set to Null if DynamicLightScattering is False and to 5 seconds if DynamicLightScattering is True.

Pattern Description: Greater than or equal to 1 second and less than or equal to 30 seconds or Null.

Programmatic Pattern: (RangeP[1*Second, 30*Second] | Automatic) | Null

AutomaticDynamicLightScatteringLaserSettings

Indicates if the DynamicLightScatteringLaserPower and DynamicLightScatteringDiodeAttenuation are automatically set at the beginning of the assay by the Instrument to levels ideal for the samples.

Default Value: Automatic

Default Calculation: Automatically set to Null if DynamicLightScattering is False. If DynamicLightScattering is True and either the DynamicLightScatteringLaserPower or DynamicLightScatteringDiodeAttenuation is specified, the option is set to False. Otherwise, AutomaticDynamicLightScatteringLaserSettings is set to True.

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

DynamicLightScatteringLaserPower

The percent of the max dynamic light scattering laser power that is used to make DLS measurements. The laser level is optimized at run time by the instrument software when AutomaticDynamicLightScatteringLaserSettings is True and LaserLevel is Null.

Default Value: Automatic

Default Calculation: Automatically set to 100% if AutomaticDynamicLightScatteringLaserSettings is False and to Null otherwise.

Pattern Description: Greater than or equal to 1 percent and less than or equal to 100 percent or Null.

Programmatic Pattern: (RangeP[1*Percent, 100*Percent] | Automatic) | Null

DynamicLightScatteringDiodeAttenuation

The percent of scattered light signal that is allowed to reach the avalanche photodiode for DLS measurements. The attenuator level is optimized at run time by the instrument software when AutomaticDynamicLightScatteringLaserSettings is True and DiodeAttenuation is Null.

Default Value: Automatic

Default Calculation: Automatically set to 100% if AutomaticDynamicLightScatteringLaserSettings is False and to Null otherwise.

Pattern Description: Greater than or equal to 1 percent and less than or equal to 100 percent or Null.

Programmatic Pattern: (RangeP[1*Percent, 100*Percent] | Automatic) | Null

Sample Storage

ContainerOut

The container which the assay samples are transferred into after the experiment.

Default Value: Null

Pattern Description: An object of type or subtype Model[Container] or Null.

Programmatic Pattern: ObjectP[Model[Container]] | Null

SamplesOutStorageCondition

The condition under which the assay samples in ContainerOut should be stored after the protocol is completed.

Default Value: Null

Pattern Description: {AmbientStorage, Refrigerator, Freezer, DeepFreezer, CryogenicStorage, YeastIncubation, YeastShakingIncubation, BacterialIncubation, BacterialShakingIncubation, MammalianIncubation, ViralIncubation, CrystalIncubation, AcceleratedTesting, IntermediateTesting, LongTermTesting, UVVisLightTesting} or Disposal or Null.

Programmatic Pattern: (SampleStorageTypeP | Disposal) | Null



Sample Prep Options

Sample Preparation

PreparatoryUnitOperations

Specifies a sequence of transferring, aliquoting, consolidating, or mixing of new or existing samples before the main experiment. These prepared samples can be used in the main experiment by referencing their defined name. For more information, please reference the documentation for ExperimentSampleManipulation.

Default Value: Null

Pattern Description: List of one or more unit Operation ManualSamplePreparation or RoboticSamplePreparation or unit Operation must match SamplePreparationP entries or Null.

Programmatic Pattern: {((ManualSamplePreparationMethodP | RoboticSamplePreparationMethodP) | SamplePreparationP)..} | Null

PreparatoryPrimitives

Specifies a sequence of transferring, aliquoting, consolidating, or mixing of new or existing samples before the main experiment. These prepared samples can be used in the main experiment by referencing their defined name. For more information, please reference the documentation for ExperimentSampleManipulation.

Default Value: Null

Pattern Description: List of one or more a primitive with head Define, Transfer, Mix, Aliquot, Consolidation, FillToVolume, Incubate, Filter, Wait, Centrifuge, or Resuspend entries or Null.

Programmatic Pattern: {SampleManipulationP..} | Null

Preparatory Incubation

Incubate

Indicates if the SamplesIn should be incubated at a fixed temperature prior to starting the experiment or any aliquoting. Sample Preparation occurs in the order of Incubation, Centrifugation, Filtration, and then Aliquoting (if specified).

Default Value: Automatic

Default Calculation: Resolves to True if any of the corresponding Incubation options are set. Otherwise, resolves to False.

Pattern Description: True or False.

Programmatic Pattern: BooleanP | Automatic

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

IncubationTemperature

Temperature at which the SamplesIn should be incubated for the duration of the IncubationTime prior to starting the experiment.

Default Value: Automatic

Pattern Description: Ambient or greater than or equal to -20 degrees Celsius and less than or equal to 500 degrees Celsius or Null.

Programmatic Pattern: ((Ambient | RangeP[$MinIncubationTemperature, $MaxIncubationTemperature]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

IncubationTime

Duration for which SamplesIn should be incubated at the IncubationTemperature, prior to starting the experiment.

Default Value: Automatic

Pattern Description: Greater than or equal to 1 minute and less than or equal to 72 hours or Null.

Programmatic Pattern: (RangeP[1*Minute, $MaxExperimentTime] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

Mix

Indicates if this sample should be mixed while incubated, prior to starting the experiment.

Default Value: Automatic

Default Calculation: Automatically resolves to True if any Mix related options are set. Otherwise, resolves to False.

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

MixType

Indicates the style of motion used to mix the sample, prior to starting the experiment.

Default Value: Automatic

Default Calculation: Automatically resolves based on the container of the sample and the Mix option.

Pattern Description: Roll, Vortex, Sonicate, Pipette, Invert, Stir, Shake, Homogenize, Swirl, Disrupt, or Nutate or Null.

Programmatic Pattern: (MixTypeP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

MixUntilDissolved

Indicates if the mix should be continued up to the MaxIncubationTime or MaxNumberOfMixes (chosen according to the mix Type), in an attempt dissolve any solute. Any mixing/incubation will occur prior to starting the experiment.

Default Value: Automatic

Default Calculation: Automatically resolves to True if MaxIncubationTime or MaxNumberOfMixes is set.

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

MaxIncubationTime

Maximum duration of time for which the samples will be mixed while incubated in an attempt to dissolve any solute, if the MixUntilDissolved option is chosen. This occurs prior to starting the experiment.

Default Value: Automatic

Default Calculation: Automatically resolves based on MixType, MixUntilDissolved, and the container of the given sample.

Pattern Description: Greater than or equal to 1 minute and less than or equal to 72 hours or Null.

Programmatic Pattern: (RangeP[1*Minute, $MaxExperimentTime] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

IncubationInstrument

The instrument used to perform the Mix and/or Incubation, prior to starting the experiment.

Default Value: Automatic

Default Calculation: Automatically resolves based on the options Mix, Temperature, MixType and container of the sample.

Pattern Description: An object of type or subtype Model[Instrument, Roller], Model[Instrument, OverheadStirrer], Model[Instrument, Vortex], Model[Instrument, Shaker], Model[Instrument, BottleRoller], Model[Instrument, Roller], Model[Instrument, Sonicator], Model[Instrument, HeatBlock], Model[Instrument, Homogenizer], Model[Instrument, Disruptor], Model[Instrument, Nutator], Model[Instrument, Thermocycler], Model[Instrument, EnvironmentalChamber], Model[Instrument, Pipette], Object[Instrument, Roller], Object[Instrument, OverheadStirrer], Object[Instrument, Vortex], Object[Instrument, Shaker], Object[Instrument, BottleRoller], Object[Instrument, Roller], Object[Instrument, Sonicator], Object[Instrument, HeatBlock], Object[Instrument, Homogenizer], Object[Instrument, Disruptor], Object[Instrument, Nutator], Object[Instrument, Thermocycler], Object[Instrument, EnvironmentalChamber], or Object[Instrument, Pipette] or Null.

Programmatic Pattern: (ObjectP[Join[MixInstrumentModels, MixInstrumentObjects]] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

AnnealingTime

Minimum duration for which the SamplesIn should remain in the incubator allowing the system to settle to room temperature after the IncubationTime has passed but prior to starting the experiment.

Default Value: Automatic

Pattern Description: Greater than or equal to 0 minutes and less than or equal to 72 hours or Null.

Programmatic Pattern: (RangeP[0*Minute, $MaxExperimentTime] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

IncubateAliquotContainer

The desired type of container that should be used to prepare and house the incubation samples which should be used in lieu of the SamplesIn for the experiment.

Default Value: Automatic

Pattern Description: An object of type or subtype Model[Container] or {Index, Container} or Null.

Programmatic Pattern: ((ObjectP[Model[Container]] | {GreaterEqualP[1, 1] | (Automatic | Null), (ObjectP[{Model[Container], Object[Container]}] | _String) | Automatic}) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

IncubateAliquotDestinationWell

The desired position in the corresponding AliquotContainer in which the aliquot samples will be placed.

Default Value: Automatic

Default Calculation: Automatically resolves to A1 in containers with only one position. For plates, fills wells in the order provided by the function AllWells.

Pattern Description: Any well from A1 to H12 or Null.

Programmatic Pattern: (WellPositionP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

IncubateAliquot

The amount of each sample that should be transferred from the SamplesIn into the IncubateAliquotContainer when performing an aliquot before incubation.

Default Value: Automatic

Default Calculation: Automatically set as the smaller between the current sample volume and the maximum volume of the destination container.

Pattern Description: All or greater than or equal to 1 microliter and less than or equal to 20 liters or Null.

Programmatic Pattern: ((RangeP[1*Microliter, 20*Liter] | All) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

Preparatory Centrifugation

Centrifuge

Indicates if the SamplesIn should be centrifuged prior to starting the experiment or any aliquoting. Sample Preparation occurs in the order of Incubation, Centrifugation, Filtration, and then Aliquoting (if specified).

Default Value: Automatic

Default Calculation: Resolves to True if any of the corresponding Centrifuge options are set. Otherwise, resolves to False.

Pattern Description: True or False.

Programmatic Pattern: BooleanP | Automatic

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

CentrifugeInstrument

The centrifuge that will be used to spin the provided samples prior to starting the experiment.

Default Value: Automatic

Pattern Description: An object of type or subtype Model[Instrument, Centrifuge] or Object[Instrument, Centrifuge] or Null.

Programmatic Pattern: (ObjectP[{Model[Instrument, Centrifuge], Object[Instrument, Centrifuge]}] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

CentrifugeIntensity

The rotational speed or the force that will be applied to the samples by centrifugation prior to starting the experiment.

Default Value: Automatic

Pattern Description: Greater than 0 revolutions per minute or greater than 0 standard accelerations due to gravity on the surface of the earth or Null.

Programmatic Pattern: ((GreaterP[0*RPM] | GreaterP[0*GravitationalAcceleration]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

CentrifugeTime

The amount of time for which the SamplesIn should be centrifuged prior to starting the experiment.

Default Value: Automatic

Pattern Description: Greater than 0 minutes or Null.

Programmatic Pattern: (GreaterP[0*Minute] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

CentrifugeTemperature

The temperature at which the centrifuge chamber should be held while the samples are being centrifuged prior to starting the experiment.

Default Value: Automatic

Pattern Description: Ambient or greater than or equal to -10 degrees Celsius and less than or equal to 40 degrees Celsius or Null.

Programmatic Pattern: ((Ambient | RangeP[-10*Celsius, 40*Celsius]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

CentrifugeAliquotDestinationWell

The desired position in the corresponding AliquotContainer in which the aliquot samples will be placed.

Default Value: Automatic

Default Calculation: Automatically resolves to A1 in containers with only one position. For plates, fills wells in the order provided by the function AllWells.

Pattern Description: Any well from A1 to H12 or Null.

Programmatic Pattern: (WellPositionP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

CentrifugeAliquotContainer

The desired type of container that should be used to prepare and house the centrifuge samples which should be used in lieu of the SamplesIn for the experiment.

Default Value: Automatic

Pattern Description: An object of type or subtype Model[Container] or {Index, Container} or Null.

Programmatic Pattern: ((ObjectP[Model[Container]] | {GreaterEqualP[1, 1] | (Automatic | Null), (ObjectP[{Model[Container], Object[Container]}] | _String) | Automatic}) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

CentrifugeAliquot

The amount of each sample that should be transferred from the SamplesIn into the CentrifugeAliquotContainer when performing an aliquot before centrifugation.

Default Value: Automatic

Default Calculation: Automatically set as the smaller between the current sample volume and the maximum volume of the destination container.

Pattern Description: All or greater than or equal to 1 microliter and less than or equal to 20 liters or Null.

Programmatic Pattern: ((RangeP[1*Microliter, 20*Liter] | All) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

Preparatory Filtering

Filtration

Indicates if the SamplesIn should be filter prior to starting the experiment or any aliquoting. Sample Preparation occurs in the order of Incubation, Centrifugation, Filtration, and then Aliquoting (if specified).

Default Value: Automatic

Default Calculation: Resolves to True if any of the corresponding Filter options are set. Otherwise, resolves to False.

Pattern Description: True or False.

Programmatic Pattern: BooleanP | Automatic

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FiltrationType

The type of filtration method that should be used to perform the filtration.

Default Value: Automatic

Default Calculation: Will automatically resolve to a filtration type appropriate for the volume of sample being filtered.

Pattern Description: PeristalticPump, Centrifuge, Vacuum, Syringe, or AirPressure or Null.

Programmatic Pattern: (FiltrationTypeP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterInstrument

The instrument that should be used to perform the filtration.

Default Value: Automatic

Default Calculation: Will automatically resolved to an instrument appropriate for the filtration type.

Pattern Description: An object of type or subtype Model[Instrument, FilterBlock], Object[Instrument, FilterBlock], Model[Instrument, PeristalticPump], Object[Instrument, PeristalticPump], Model[Instrument, VacuumPump], Object[Instrument, VacuumPump], Model[Instrument, Centrifuge], Object[Instrument, Centrifuge], Model[Instrument, SyringePump], or Object[Instrument, SyringePump] or Null.

Programmatic Pattern: (ObjectP[{Model[Instrument, FilterBlock], Object[Instrument, FilterBlock], Model[Instrument, PeristalticPump], Object[Instrument, PeristalticPump], Model[Instrument, VacuumPump], Object[Instrument, VacuumPump], Model[Instrument, Centrifuge], Object[Instrument, Centrifuge], Model[Instrument, SyringePump], Object[Instrument, SyringePump]}] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

Filter

The filter that should be used to remove impurities from the SamplesIn prior to starting the experiment.

Default Value: Automatic

Default Calculation: Will automatically resolve to a filter appropriate for the filtration type and instrument.

Pattern Description: An object of type or subtype Model[Container, Plate, Filter], Model[Container, Vessel, Filter], or Model[Item, Filter] or Null.

Programmatic Pattern: (ObjectP[{Model[Container, Plate, Filter], Model[Container, Vessel, Filter], Model[Item, Filter]}] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterMaterial

The membrane material of the filter that should be used to remove impurities from the SamplesIn prior to starting the experiment.

Default Value: Automatic

Default Calculation: Resolves to an appropriate filter material for the given sample is Filtration is set to True.

Pattern Description: Cellulose, Cotton, Polyethylene, Polypropylene, PTFE, Nylon, PES, PLUS, PVDF, GlassFiber, GHP, UHMWPE, EPDM, DuraporePVDF, GxF, ZebaDesaltingResin, NickelResin, Silica, or HLB or Null.

Programmatic Pattern: (FilterMembraneMaterialP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

PrefilterMaterial

The material from which the prefilter filtration membrane should be made of to remove impurities from the SamplesIn prior to starting the experiment.

Default Value: Automatic

Default Calculation: By default, no prefiltration is performed on samples, even when Filter->True.

Pattern Description: Cellulose, Cotton, Polyethylene, Polypropylene, PTFE, Nylon, PES, PLUS, PVDF, GlassFiber, GHP, UHMWPE, EPDM, DuraporePVDF, GxF, ZebaDesaltingResin, NickelResin, Silica, or HLB or Null.

Programmatic Pattern: (FilterMembraneMaterialP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterPoreSize

The pore size of the filter that should be used when removing impurities from the SamplesIn prior to starting the experiment.

Default Value: Automatic

Default Calculation: Resolves to an appropriate filter pore size for the given sample is Filtration is set to True.

Pattern Description: 0.008 micrometers, 0.1 micrometers, 0.2 micrometers, 0.22 micrometers, 0.45 micrometers, 1. micrometer, 1.1 micrometers, 2.5 micrometers, 6. micrometers, 20. micrometers, 30. micrometers, or 100. micrometers or Null.

Programmatic Pattern: (FilterSizeP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

PrefilterPoreSize

The pore size of the filter; all particles larger than this should be removed during the filtration.

Default Value: Automatic

Default Calculation: By default, no prefiltration is performed on samples, even when Filter->True.

Pattern Description: 0.008 micrometers, 0.1 micrometers, 0.2 micrometers, 0.22 micrometers, 0.45 micrometers, 1. micrometer, 1.1 micrometers, 2.5 micrometers, 6. micrometers, 20. micrometers, 30. micrometers, or 100. micrometers or Null.

Programmatic Pattern: (FilterSizeP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterSyringe

The syringe used to force that sample through a filter.

Default Value: Automatic

Default Calculation: Resolves to an syringe appropriate to the volume of sample being filtered, if Filtration is set to True.

Pattern Description: An object of type or subtype Model[Container, Syringe] or Object[Container, Syringe] or a prepared sample or Null.

Programmatic Pattern: ((ObjectP[{Model[Container, Syringe], Object[Container, Syringe]}] | _String) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterHousing

The filter housing that should be used to hold the filter membrane when filtration is performed using a standalone filter membrane.

Default Value: Automatic

Default Calculation: Resolve to an housing capable of holding the size of the membrane being used, if filter with Membrane FilterType is being used and Filtration is set to True.

Pattern Description: An object of type or subtype Model[Instrument, FilterHousing], Object[Instrument, FilterHousing], Model[Instrument, FilterBlock], or Object[Instrument, FilterBlock] or Null.

Programmatic Pattern: (ObjectP[{Model[Instrument, FilterHousing], Object[Instrument, FilterHousing], Model[Instrument, FilterBlock], Object[Instrument, FilterBlock]}] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterIntensity

The rotational speed or force at which the samples will be centrifuged during filtration.

Default Value: Automatic

Default Calculation: Will automatically resolve to 2000 GravitationalAcceleration if FiltrationType is Centrifuge and Filtration is True.

Pattern Description: Greater than 0 revolutions per minute or greater than 0 standard accelerations due to gravity on the surface of the earth or Null.

Programmatic Pattern: ((GreaterP[0*RPM] | GreaterP[0*GravitationalAcceleration]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterTime

The amount of time for which the samples will be centrifuged during filtration.

Default Value: Automatic

Default Calculation: Will automatically resolve to 5 Minute if FiltrationType is Centrifuge and Filtration is True.

Pattern Description: Greater than 0 minutes or Null.

Programmatic Pattern: (GreaterP[0*Minute] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterTemperature

The temperature at which the centrifuge chamber will be held while the samples are being centrifuged during filtration.

Default Value: Automatic

Default Calculation: Will automatically resolve to 22 Celsius if FiltrationType is Centrifuge and Filtration is True.

Pattern Description: Greater than or equal to 4 degrees Celsius or Null.

Programmatic Pattern: ((Alternatives[GreaterEqualP[4*Celsius]]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterContainerOut

The desired container filtered samples should be produced in or transferred into by the end of filtration, with indices indicating grouping of samples in the same plates, if desired.

Default Value: Automatic

Default Calculation: Automatically set as the PreferredContainer for the Volume of the sample. For plates, attempts to fill all wells of a single plate with the same model before using another one.

Pattern Description: An object of type or subtype Model[Container] or Object[Container] or a prepared sample or {Index, Container} or Null.

Programmatic Pattern: (((ObjectP[{Model[Container], Object[Container]}] | _String) | {GreaterEqualP[1, 1] | Automatic, (ObjectP[{Model[Container], Object[Container]}] | _String) | Automatic}) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterAliquotDestinationWell

The desired position in the corresponding AliquotContainer in which the aliquot samples will be placed.

Default Value: Automatic

Default Calculation: Automatically resolves to A1 in containers with only one position. For plates, fills wells in the order provided by the function AllWells.

Pattern Description: Any well from A1 to H12 or Null.

Programmatic Pattern: (WellPositionP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterAliquotContainer

The desired type of container that should be used to prepare and house the filter samples which should be used in lieu of the SamplesIn for the experiment.

Default Value: Automatic

Pattern Description: An object of type or subtype Model[Container] or {Index, Container} or Null.

Programmatic Pattern: ((ObjectP[Model[Container]] | {GreaterEqualP[1, 1] | (Automatic | Null), (ObjectP[{Model[Container], Object[Container]}] | _String) | Automatic}) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterAliquot

The amount of each sample that should be transferred from the SamplesIn into the FilterAliquotContainer when performing an aliquot before filtration.

Default Value: Automatic

Default Calculation: Automatically set as the smaller between the current sample volume and the maximum volume of the destination container.

Pattern Description: All or greater than or equal to 1 microliter and less than or equal to 20 liters or Null.

Programmatic Pattern: ((RangeP[1*Microliter, 20*Liter] | All) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

FilterSterile

Indicates if the filtration of the samples should be done in a sterile environment.

Default Value: Automatic

Default Calculation: Resolve to False if Filtration is indicated. If sterile filtration is desired, this option must manually be set to True.

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

Aliquoting

Aliquot

Indicates if aliquots should be taken from the SamplesIn and transferred into new AliquotSamples used in lieu of the SamplesIn for the experiment. Note that if NumberOfReplicates is specified this indicates that the input samples will also be aliquoted that number of times. Note that Aliquoting (if specified) occurs after any Sample Preparation (if specified).

Figure 3.5: Prior to measurement in ExperimentThermalShift any number of samples may be pooled to create new assay samples with the following steps. Step1: Samples are pooled in liquid handler compatible AliquotContainer. Step2: After pooling, pooled samples can optionally be mixed either by pipetting or vortexing. Step3: After pooling and mixing (if desired), pooled samples can be optionally incubated at a desired temperature for a period of time. Step4: After incubation, pooled samples can remain in the thermal incubation instrument before being removed, allowing the system to settle to room temperature.

Default Value: Automatic

Pattern Description: True or False.

Programmatic Pattern: BooleanP | Automatic

Index Matches to: experiment samples

AliquotAmount

The amount of a sample that should be transferred from the input samples into aliquots.

Default Value: Automatic

Default Calculation: Automatically set as the smaller between the current sample volume and the maximum volume of the destination container if a liquid, or the current Mass or Count if a solid or counted item, respectively.

Pattern Description: All or Count or Count or Mass or Volume or Null.

Programmatic Pattern: ((RangeP[1*Microliter, 20*Liter] | RangeP[1*Milligram, 20*Kilogram] | GreaterP[0*Unit, 1*Unit] | GreaterP[0., 1.] | All) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

TargetConcentration

The desired final concentration of analyte in the AliquotSamples after dilution of aliquots of SamplesIn with the ConcentratedBuffer and BufferDiluent which should be used in lieu of the SamplesIn for the experiment.

Default Value: Automatic

Default Calculation: Automatically calculated based on aliquot and buffer volumes.

Pattern Description: Greater than 0 molar or greater than 0 grams per liter or Null.

Programmatic Pattern: ((GreaterP[0*Molar] | GreaterP[0*(Gram/Liter)]) | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

TargetConcentrationAnalyte

The substance whose final concentration is attained with the TargetConcentration option.

Default Value: Automatic

Default Calculation: Automatically set to the first value in the Analytes field of the input sample, or, if not populated, to the first analyte in the Composition field of the input sample, or if none exist, the first identity model of any kind in the Composition field.

Pattern Description: An object of type or subtype Model[Molecule], Model[Molecule, cDNA], Model[Molecule, Oligomer], Model[Molecule, Transcript], Model[Molecule, Protein], Model[Molecule, Protein, Antibody], Model[Molecule, Carbohydrate], Model[Molecule, Polymer], Model[Resin], Model[Resin, SolidPhaseSupport], Model[Lysate], Model[ProprietaryFormulation], Model[Virus], Model[Cell], Model[Cell, Mammalian], Model[Cell, Bacteria], Model[Cell, Yeast], Model[Tissue], Model[Material], or Model[Species] or Null.

Programmatic Pattern: (ObjectP[IdentityModelTypes] | Automatic) | Null

Index Matches to: experiment samples

Nested Index Matches to: experiment samples

AssayVolume

The desired total volume of the aliquoted sample plus dilution buffer.

Default Value: Automatic

Default Calculation: Automatically determined based on Volume and TargetConcentration option values.

Pattern Description: Greater than or equal to 1 microliter and less than or equal to 20 liters or Null.

Programmatic Pattern: (RangeP[1*Microliter, 20*Liter] | Automatic) | Null

Index Matches to: experiment samples

ConcentratedBuffer

The concentrated buffer which should be diluted by the BufferDilutionFactor in the final solution (i.e., the combination of the sample, ConcentratedBuffer, and BufferDiluent). The ConcentratedBuffer and BufferDiluent will be combined and then mixed with the sample, where the combined volume of these buffers is the difference between the AliquotAmount and the total AssayVolume.

Default Value: Automatic

Pattern Description: An object of type or subtype Model[Sample] or Object[Sample] or a prepared sample or Null.

Programmatic Pattern: ((ObjectP[{Model[Sample], Object[Sample]}] | _String) | Automatic) | Null

Index Matches to: experiment samples

BufferDilutionFactor

The dilution factor by which the concentrated buffer should be diluted in the final solution (i.e., the combination of the sample, ConcentratedBuffer, and BufferDiluent). The ConcentratedBuffer and BufferDiluent will be combined and then mixed with the sample, where the combined volume of these buffers is the difference between the AliquotAmount and the total AssayVolume.

Default Value: Automatic

Default Calculation: If ConcentratedBuffer is specified, automatically set to the ConcentratedBufferDilutionFactor of that sample; otherwise, set to Null.

Pattern Description: Greater than or equal to 1 in increments of 1 or Null.

Programmatic Pattern: (GreaterEqualP[1, 1] | Automatic) | Null

Index Matches to: experiment samples

BufferDiluent

The buffer used to dilute the aliquot sample such that ConcentratedBuffer is diluted by BufferDilutionFactor in the final solution. The ConcentratedBuffer and BufferDiluent will be combined and then mixed with the sample, where the combined volume of these buffers is the difference between the AliquotAmount and the total AssayVolume.

Default Value: Automatic

Default Calculation: Automatically resolves to Model[Sample, "Milli-Q water"] if ConcentratedBuffer is specified; otherwise, resolves to Null.

Pattern Description: An object of type or subtype Model[Sample] or Object[Sample] or a prepared sample or Null.

Programmatic Pattern: ((ObjectP[{Model[Sample], Object[Sample]}] | _String) | Automatic) | Null

Index Matches to: experiment samples

AssayBuffer

The buffer that should be added to any aliquots requiring dilution, where the volume of this buffer added is the difference between the AliquotAmount and the total AssayVolume.

Default Value: Automatic

Default Calculation: Automatically resolves to Model[Sample, "Milli-Q water"] if ConcentratedBuffer is not specified; otherwise, resolves to Null.

Pattern Description: An object of type or subtype Model[Sample] or Object[Sample] or a prepared sample or Null.

Programmatic Pattern: ((ObjectP[{Model[Sample], Object[Sample]}] | _String) | Automatic) | Null

Index Matches to: experiment samples

AliquotSampleStorageCondition

The non-default conditions under which any aliquot samples generated by this experiment should be stored after the protocol is completed.

Default Value: Automatic

Pattern Description: {AmbientStorage, Refrigerator, Freezer, DeepFreezer, CryogenicStorage, YeastIncubation, YeastShakingIncubation, BacterialIncubation, BacterialShakingIncubation, MammalianIncubation, ViralIncubation, CrystalIncubation, AcceleratedTesting, IntermediateTesting, LongTermTesting, UVVisLightTesting} or Disposal or Null.

Programmatic Pattern: ((SampleStorageTypeP | Disposal) | Automatic) | Null

Index Matches to: experiment samples

DestinationWell

The desired position in the corresponding AliquotContainer in which the aliquot samples will be placed.

Default Value: Automatic

Default Calculation: Automatically resolves to A1 in containers with only one position. For plates, fills wells in the order provided by the function AllWells.

Pattern Description: Any well from A1 to H12 or list of one or more any well from A1 to H12 or any well from A1 to H12 entries or Null.

Programmatic Pattern: ((WellPositionP | {((Automatic | Null) | WellPositionP)..}) | Automatic) | Null

AliquotContainer

The desired type of container that should be used to prepare and house the aliquot samples, with indices indicating grouping of samples in the same plates, if desired. This option will resolve to be the length of the SamplesIn * NumberOfReplicates.

Default Value: Automatic

Default Calculation: Automatically set as the PreferredContainer for the AssayVolume of the sample. For plates, attempts to fill all wells of a single plate with the same model before aliquoting into the next.

Pattern Description: An object of type or subtype Model[Container] or Object[Container] or a prepared sample or Automatic or Null or {Index, Container} or list of one or more an object of type or subtype Model[Container] or Object[Container] or a prepared sample or Automatic or Null entries or list of one or more Automatic or Null or {Index, Container} entries.

Programmatic Pattern: (((ObjectP[{Model[Container], Object[Container]}] | _String) | (Automatic | Null) | {GreaterEqualP[1, 1] | (Automatic | Null), (ObjectP[{Model[Container], Object[Container]}] | _String) | (Automatic | Null)} | {((ObjectP[{Model[Container], Object[Container]}] | _String) | (Automatic | Null))..} | {({GreaterEqualP[1, 1] | (Automatic | Null), (ObjectP[{Model[Container], Object[Container]}] | _String) | (Automatic | Null)} | (Automatic | Null))..}) | Automatic) | Null

AliquotPreparation

Indicates the desired scale at which liquid handling used to generate aliquots will occur.

Default Value: Automatic

Default Calculation: Automatic resolution will occur based on manipulation volumes and container types.

Pattern Description: Manual or Robotic or Null.

Programmatic Pattern: (PreparationMethodP | Automatic) | Null

ConsolidateAliquots

Indicates if identical aliquots should be prepared in the same container/position.

Default Value: Automatic

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null



Protocol Options

Organizational Information

Template

A template protocol whose methodology should be reproduced in running this experiment. Option values will be inherited from the template protocol, but can be individually overridden by directly specifying values for those options to this Experiment function.

Default Value: Null

Pattern Description: An object of type or subtype Object[Protocol] or an object of type or subtype of Object[Protocol] with UnresolvedOptions, ResolvedOptions specified or Null.

Programmatic Pattern: (ObjectP[Object[Protocol]] | FieldReferenceP[Object[Protocol], {UnresolvedOptions, ResolvedOptions}]) | Null

Name

A object name which should be used to refer to the output object in lieu of an automatically generated ID number.

Default Value: Null

Pattern Description: A string or Null.

Programmatic Pattern: _String | Null

Post Experiment

MeasureWeight

Indicates if any solid samples that are modified in the course of the experiment should have their weights measured and updated after running the experiment. Please note that public samples are weighed regardless of the value of this option.

Default Value: Automatic

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

MeasureVolume

Indicates if any liquid samples that are modified in the course of the experiment should have their volumes measured and updated after running the experiment. Please note that public samples are volume measured regardless of the value of this option.

Default Value: Automatic

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null

ImageSample

Indicates if any samples that are modified in the course of the experiment should be freshly imaged after running the experiment. Please note that public samples are imaged regardless of the value of this option.

Default Value: Automatic

Pattern Description: True or False or Null.

Programmatic Pattern: (BooleanP | Automatic) | Null



Example Calls

Linear and Stepped Thermal Profiles

Indicate the rate of temperature change during a linear thermal profile from 25 Celsius to 95 Celsius.

ExperimentThermalShift[mySamples,

 	TemperatureRampRate -> (0.1 * Celsius / Second),

 	TemperatureRamping -> Linear,

 	MinTemperature -> (25 * Celsius),

 	MaxTemperature -> (95 * Celsius)

 ]

Indicate the rate of temperature change during a stepped thermal profile from 30 Celsius to 90 Celsius with measurements every 2.5 degrees.

ExperimentThermalShift[mySamples,

 	TemperatureRampRate -> (0.1 * Celsius / Second),

 	TemperatureRamping -> Step, NumberOfTemperatureRampSteps -> 24,

 MinTemperature -> (30 * Celsius),

 	MaxTemperature -> (90 * Celsius)

 ]

Indicate data collection over 2 {Heating, Cooling} cycles.

ExperimentThermalShift[mySamples,

 	TemperatureRampOrder -> {Heating, Cooling},

 	NumberOfCycles -> 2

 ]

Singleton and pooled samples

Generate a thermal shift protocol to analyze a list of singleton sample inputs:

ExperimentThermalShift[{mySample1, mySample2}]

Generates a thermal shift protocol from a nested list of inputs by pooling inputs within the same nested list:

ExperimentThermalShift[

 	{{mySample1, mySample2}, {mySample3, mySample4}},

 	AliquotAmount -> {

   		{50 * Microliter, 50 * Microliter},

   		{50 * Microliter, 50 * Microliter}

   	},

 	AliquotContainer ->

  Model[Container, Plate, "96-well 2mL Deep Well Plate"]

 ]

Generates a thermal shift protocol from a list of singleton containers by treating each well inside the container as an independent input sample:

ExperimentThermalShift[{myContainer}]

Generates a thermal shift protocol from a nested list of containers by pooling all the samples in the container:

ExperimentThermalShift[{{myContainer}},

 	AliquotAmount -> (50 * Microliter),

 	AliquotContainer ->

  Model[Container, Plate, "96-well 2mL Deep Well Plate"]

 ]

Dilution and Serial Dilution Curves

Indicate the type and number of serial dilutions of a single sample. Provide the volume of the sample that should be transferred, the volume of diluent the sample should be added to and the number of times the sample should be diluted. This should be used if you want give the volumes the sample will be diluted by with instead of dilution factors. The example call below will create a series of samples with dilution factors of (0.5, 0.25, 0.125, 0.0625, 0.03125) and a volume of 25 Microliters.

ExperimentThermalShift[mySample,

 	SerialDilutionCurve -> {{25 * Microliter, 25 * Microliter, 5}}

 ]

Generate a serial dilution curve to dilute the sample. Provide the volume of the sample, a dilution factor and the number of times the sample should be diluted. This should be used if you want the sample to be diluted by the same factor at each step of the serial dilution. The volume of the sample will be the final volume of the sample after the dilution is performed. The example call below will create a series of samples with dilution factors of (0.7, 0.49, 0.34, 0.24, 0.17, 0.12) and a volume of 60 Microliters.

ExperimentThermalShift[mySamples,

 SerialDilutionCurve -> {60 * Microliter, {0.7, 6}}]

Generate a serial dilution curve to dilute the sample. Provide the volume of the sample and a series of dilution factors the sample should be diluted by at each step. This should be used if you want the sample to be diluted by different factors at each step of the serial dilution. The volume of the sample will be the final volume of the sample after the dilution is performed. The example call below will create a series of samples with dilution factors of (0.7, 0.49, 0.34, 0.31, 0.28, 0.25) and a volume of 60 Microliters.

ExperimentThermalShift[mySamples,

 	SerialDilutionCurve -> {60 * Microliter, {0.7, 0.7, 0.7, 0.9, 0.9,

    0.9}}

 ]

Indicate the type and number of dilutions of a single sample.

ExperimentThermalShift[mySample,

 	DilutionCurve -> {

   		{

    			{25 * Microliter, 25 * Microliter},

    			{25 * Microliter, 50 * Microliter},

    			{25 * Microliter, 100 * Microliter}

    		}

   	}

 ]

Indicate the type and number of serial dilutions of each sample within a pooled sample.

ExperimentThermalShift[{{mySample1, mySample2}},

 	SerialDilutionCurve -> {

   		{

    			{25 * Microliter, 25 * Microliter, 5},

    			{25 * Microliter, 50 * Microliter, 5}

    		}

   	},

 	AliquotAmount -> {{20 * Microliter, 20 * Microliter}},

 	AliquotContainer ->

  Model[Container, Plate, "96-well 2mL Deep Well Plate"]

 ]

Indicate the type and number of dilutions of each sample within a pooled sample.

ExperimentThermalShift[{{mySample1, mySample2}},

 	DilutionCurve -> {

   		{

    			{

     				{25 * Microliter, 25 * Microliter},

     				{25 * Microliter, 50 * Microliter},

     				{25 * Microliter, 100 * Microliter}

     			},

    			{

     				{25 * Microliter, 25 * Microliter},

     				{25 * Microliter, 25 * Microliter},

     				{25 * Microliter, 25 * Microliter}

     			}

    		}

   	},

 	AliquotAmount -> {{50 * Microliter, 50 * Microliter}},

 	AliquotContainer ->

  Model[Container, Plate, "96-well 2mL Deep Well Plate"]

 ]



Preferred Input Containers

The ViiA 7 accepts MicroAmp EnduraPlate optical 384-well clear reaction plates.

The Uncle accepts capillary strips.

[end context 1]

Here are summaries of the previous meetings:

[begin summary 1]

To further refine the implementation of the **ExperimentThermalShift** protocol and address the Scientific Critic's feedback, I will enhance the clarity of justifications, provide more specific examples, and simplify explanations where possible.

### Revised ExperimentThermalShift Setup

```plaintext
ExperimentThermalShift[
  myPeptideSamples,
  Instrument -> Model[Instrument, MultimodeSpectrophotometer, "Uncle"],
  NumberOfReplicates -> 3,
  TemperatureRampRate -> (0.5 * Celsius / Minute),
  MinTemperature -> (25 * Celsius),
  MaxTemperature -> (85 * Celsius),
  NumberOfCycles -> 1,
  OptimizeFluorescenceLaserPower -> True
]
```

### Enhanced Explanation of Specified Options

1. **Instrument**
   - **Default Value**: Automatic (set to Model[Instrument, MultimodeSpectrophotometer, "Uncle"] for protein samples)
   - **Chosen Value**: Model[Instrument, MultimodeSpectrophotometer, "Uncle"]
   - **Rationale**: The Uncle spectrophotometer is ideal for detecting subtle conformational changes and aggregation in peptides due to its dual measurement capabilities. For example, Smith et al. (2020) demonstrated its effectiveness in analyzing peptide stability by showing how it accurately detects changes in fluorescence and light scattering, which are critical for understanding peptide behavior under thermal stress.

2. **NumberOfReplicates**
   - **Default Value**: Automatic (set to 2 for MultimodeSpectrophotometer)
   - **Chosen Value**: 3
   - **Rationale**: Using three replicates increases data reliability and reduces variability, as supported by Jones et al. (2019), who found that three replicates provide robust statistical power for detecting significant differences in peptide stability. This ensures that our findings are statistically sound and reproducible.

3. **TemperatureRampRate**
   - **Default Value**: Automatic (set to 1 Celsius/Minute for Linear)
   - **Chosen Value**: 0.5 Celsius/Minute
   - **Rationale**: A slower ramp rate improves the resolution of thermal transitions, allowing for more precise detection of melting points. Brown et al. (2018) highlighted that slower rates are beneficial for capturing detailed thermal profiles, which is essential for accurately assessing peptide stability.

4. **MinTemperature and MaxTemperature**
   - **Default Values**: 15 degrees Celsius and 95 degrees Celsius
   - **Chosen Values**: 25 degrees Celsius and 85 degrees Celsius
   - **Rationale**: These temperatures are chosen to reflect physiological conditions and prevent peptide denaturation. Lee et al. (2017) found that these settings are optimal for capturing relevant thermal transitions without compromising peptide integrity, ensuring that our peptides are stable under conditions similar to those in the human body.

5. **NumberOfCycles**
   - **Default Value**: Automatic (set to 0.5 cycles for ViiA 7)
   - **Chosen Value**: 1
   - **Rationale**: A full cycle (heating and cooling) provides a comprehensive thermal profile, capturing reversible transitions. This is crucial for assessing both stability and potential aggregation of peptides, ensuring that we have a complete understanding of their thermal behavior.

6. **OptimizeFluorescenceLaserPower**
   - **Default Value**: False
   - **Chosen Value**: True
   - **Rationale**: Optimizing the fluorescence laser power ensures the signal is within the optimal range for detection. We will adjust the laser power to achieve a target emission intensity range of 5% to 20% of the detector's saturation intensity, ensuring linearity and accuracy in measurements.

### Integration with Machine Learning

The data from the ExperimentThermalShift protocol will be integrated into machine learning models to predict peptide stability and binding affinity. We will use features such as melting temperature and aggregation data to refine models using neural networks and support vector machines. This integration forms a feedback loop where machine learning predictions guide experimental setups, and experimental results refine computational models. For example, data on thermal stability will inform the prediction of peptide modifications that enhance stability, directly contributing to the development of therapeutics.

### Connection to Clinical Value

The ExperimentThermalShift protocol contributes to clinical value by ensuring that peptide candidates have the necessary thermal stability for therapeutic use. Stable peptides are more likely to maintain efficacy in physiological conditions, which is crucial for developing effective treatments for diseases such as Type 2 Diabetes and Alzheimer's. By focusing on stability, we enhance the likelihood of developing therapeutics that address unmet medical needs, ultimately benefiting patients.

### Risk Assessment and Mitigation

Potential risks include sample degradation or unexpected aggregation. To mitigate these risks, we will implement strict sample handling and storage protocols, such as maintaining samples at low temperatures until analysis. If initial protocols do not prevent degradation, we will adjust storage conditions or explore alternative sample preparation methods to ensure sample integrity.

### Conclusion

This revised setup and explanation aim to provide a comprehensive and scientifically rigorous approach to implementing the ExperimentThermalShift protocol. By addressing the feedback, we ensure that the experimental design is robust, well-justified, and integrated with the broader research objectives, ultimately contributing to high clinical value and impactful application of machine learning.

[end summary 1]

[begin summary 2]

To further refine the implementation of the ExperimentThermalShift protocol and address the Scientific Critic's feedback, I will provide more detailed justifications and ensure clarity in how the experiment aligns with our project goals. Here is the revised plan:

### Revised Experiment Implementation:

```plaintext
ExperimentThermalShift[
  mySamples,
  Instrument -> Model[Instrument, MultimodeSpectrophotometer, "Uncle"],
  NumberOfCycles -> 2,
  TemperatureRampRate -> (0.1 * Celsius / Second),
  TemperatureRamping -> Linear,
  MinTemperature -> (25 * Celsius),
  MaxTemperature -> (95 * Celsius),
  DynamicLightScattering -> True,
  DynamicLightScatteringMeasurements -> {Before, After}
]
```

### Detailed Justifications and Improvements:

1. **Integration with Machine Learning**:
   - **Machine Learning Models**: We are employing neural networks to predict peptide stability and binding affinity. The thermal shift data, specifically the melting temperature (Tm) and aggregation behavior, will be used as input features in these models. This data will help refine the models by providing empirical evidence of stability, which is crucial for designing peptides with enhanced therapeutic potential. The integration will involve adjusting model parameters based on observed stability metrics, thereby improving prediction accuracy.

2. **Instrument Choice Justification**:
   - **Uncle Instrument**: The Uncle's ability to measure both fluorescence and static light scattering provides comprehensive data on peptide stability and aggregation. We will analyze changes in fluorescence intensity and scattering intensity to assess conformational changes and aggregation states. For example, a decrease in fluorescence intensity or an increase in scattering intensity could indicate peptide unfolding or aggregation, respectively. These metrics will inform decisions on peptide modifications to enhance stability.

3. **Temperature Range and Ramp Rate**:
   - **Empirical Support**: The chosen temperature range (25°C to 95°C) and ramp rate (0.1°C/second) are based on preliminary studies and literature that indicate these conditions effectively capture the thermal transitions of similar peptides. For instance, studies have shown that peptides with therapeutic potential often exhibit key stability transitions within this range, making it ideal for our analysis.

4. **Dynamic Light Scattering (DLS) Measurements**:
   - **Aggregation Data Utilization**: DLS data will provide insights into the aggregation state of peptides, which is critical for understanding their solubility and stability. Changes in aggregation state will guide decisions on peptide modifications, such as altering sequence or formulation, to improve therapeutic efficacy. For example, if significant aggregation is observed, we may consider introducing sequence modifications to reduce hydrophobic interactions.

5. **Decision Points and Data Utilization**:
   - **Criteria and Contingency Plans**: Peptides with a Tm above 50°C and minimal aggregation will proceed to further validation. If a peptide does not meet these criteria, we will revisit the design, potentially modifying the sequence or employing chemical modifications to enhance stability. This iterative process ensures that only the most promising candidates advance. Specific strategies may include cyclization to enhance stability or the introduction of non-natural amino acids to reduce degradation.

6. **Simplification and Clarity**:
   - **Iterative Process Explanation**: The iterative process between experimental data and machine learning models involves using the thermal shift data to update model parameters, improving their predictive power. This feedback loop ensures continuous refinement of peptide designs based on empirical evidence. Key steps include data collection, model adjustment, and design iteration.

7. **Adherence to Agenda**:
   - All specified options are directly tied to the experimental goals and constraints outlined in the agenda. The choices are justified within the context of enhancing the clinical value and scientific impact of the project.

By incorporating these improvements, the experiment is better aligned with the agenda's objectives, ensuring that it contributes to the project's clinical value and scientific impact. This approach also ensures that the data obtained will be effectively integrated into our machine learning models, enhancing their utility in peptide design.

[end summary 2]

[begin summary 3]

To further refine the response and ensure it fully addresses the feedback from the Scientific Critic, I will provide additional details and clarifications that align with the agenda's goals.

### Revised Experiment Setup

```plaintext
ExperimentThermalShift[
    myPeptideSamples,
    Instrument -> Model[Instrument, MultimodeSpectrophotometer, "Uncle"],
    NumberOfCycles -> 2,
    TemperatureRampRate -> (0.1 * Celsius / Second),
    TemperatureRamping -> Linear,
    MinTemperature -> (25 * Celsius),
    MaxTemperature -> (95 * Celsius),
    DynamicLightScattering -> True
]
```

### Detailed Justifications and Integration

1. **Integration with Machine Learning**:
   - **Feature Utilization**: The extracted features, such as melting temperature (Tm) and aggregation onset, will be used to train machine learning models that predict peptide stability and binding affinity. These features will serve as inputs to algorithms like random forests and neural networks, which will be trained to identify patterns and correlations that inform peptide design. For instance, a higher Tm might correlate with increased stability, guiding the selection of peptide modifications that enhance this property.

2. **Instrument Choice Justification**:
   - **Uncle Spectrophotometer**: The choice of the Uncle spectrophotometer is based on its proven efficacy in similar studies, such as those analyzing monoclonal antibodies and protein stability. For example, a study by Smith et al. (2020) demonstrated its capability to accurately measure protein aggregation and stability, which is directly applicable to our peptide analysis.

3. **Number of Cycles**:
   - **Supporting Data**: The decision to use 2 cycles is supported by preliminary data from our lab, which showed that multiple cycles can capture reversible structural changes in peptides. Additionally, literature such as Johnson et al. (2019) highlights the importance of multiple thermal cycles in revealing transient states in protein folding.

4. **Temperature Ramp Rate**:
   - **Trade-offs and Limitations**: The selected ramp rate of 0.1 Celsius/Second provides a balance between detailed data collection and experiment duration. While slower rates offer higher resolution, they also extend the experiment time. This rate was chosen to ensure comprehensive analysis without exceeding the 3-month project timeline.

5. **Dynamic Light Scattering**:
   - **Validation and Cross-Referencing**: Parameters such as hydrodynamic radius and polydispersity index will be validated against other experimental data, such as circular dichroism results, to ensure accuracy. This cross-referencing will help confirm the reliability of the aggregation data and its integration into machine learning models.

6. **Project Timeline and Risk Management**:
   - **Specific Scenarios and Adjustments**: If unexpected aggregation occurs, we will adjust peptide formulations by altering buffer conditions or peptide concentrations. These adjustments will be made swiftly to minimize impact on the timeline. Regular data reviews will facilitate early detection of issues, allowing for prompt corrective actions.

7. **Adherence to Agenda**:
   - **Connection to Project Goals**: Each experimental decision is explicitly linked to the project's objectives of high clinical value and impactful machine learning application. By ensuring that our peptide candidates are stable and effective, we contribute to the development of therapeutics with significant clinical potential. This alignment ensures that our work not only advances scientific understanding but also addresses unmet medical needs.

By incorporating these detailed justifications and strategies, the experiment setup is now more thoroughly aligned with the project's strategic goals, providing a clear path for integrating experimental data into our machine learning-driven drug discovery process.

[end summary 3]

[begin summary 4]

To further refine the implementation of the `ExperimentThermalShift` protocol and address the feedback, I'll provide more explicit details on the integration with machine learning, clarify the clinical value, and enhance the decision points and risk management strategies.

### Revised Experiment Setup

```plaintext
ExperimentThermalShift[
    mySamples,
    Instrument -> Model[Instrument, MultimodeSpectrophotometer, "Uncle"],
    NumberOfCycles -> 2,
    TemperatureRampRate -> (0.1 * Celsius / Second),
    TemperatureRamping -> Linear,
    MinTemperature -> (25 * Celsius),
    MaxTemperature -> (95 * Celsius),
    DynamicLightScattering -> True
]
```

### Enhanced Justification and Connection to Agenda Goals

1. **Instrument Choice**
   - **Justification**: The Uncle spectrophotometer is selected for its dual capability to measure fluorescence and static light scattering, providing comprehensive data on both thermal stability and aggregation. This is crucial for understanding peptide behavior under thermal stress.
   - **Connection to Goals**: High-resolution data from the Uncle will directly inform machine learning models, improving predictions of peptide stability and guiding further design iterations.

2. **Number of Cycles**
   - **Justification**: Two cycles ensure data reproducibility, allowing us to confirm consistent thermal stability across multiple runs.
   - **Connection to Goals**: Reliable data is essential for validating machine learning predictions and refining peptide designs, contributing to the project's clinical value.

3. **Temperature Ramp Rate**
   - **Justification**: A ramp rate of 0.1 Celsius/Second is chosen to capture subtle changes in peptide stability, balancing experimental duration with resolution.
   - **Connection to Goals**: Precise Tm determination is vital for assessing peptide stability, a key factor in therapeutic efficacy and a critical input for machine learning models.

4. **Temperature Range**
   - **Justification**: Starting at 25°C minimizes environmental noise, while 95°C captures the full range of thermal transitions for proteins.
   - **Connection to Goals**: Understanding the full thermal profile aids in optimizing peptide stability, crucial for therapeutic potential.

5. **Dynamic Light Scattering (DLS)**
   - **Justification**: DLS provides insights into peptide aggregation, informing design strategies to minimize undesirable aggregation.
   - **Connection to Goals**: Aggregation data enhances machine learning models, improving predictions of peptide behavior and therapeutic viability.

6. **Integration with Machine Learning**
   - **Explanation**: Thermal shift data, including Tm and aggregation behavior, will refine machine learning models by providing real-world validation of predicted stability and binding affinity. We will employ neural networks to predict peptide stability and support vector machines for binding affinity predictions. Feedback loops will occur bi-weekly, allowing for timely model updates and design adjustments.
   - **Connection to Goals**: This integration ensures effective use of machine learning, enhancing the project's scientific impact and clinical relevance.

7. **Decision Points and Criteria**
   - **Explanation**: Success criteria include achieving a Tm above 50°C and minimal aggregation at physiological temperatures. Peptides meeting these benchmarks will advance to further development stages. These thresholds are based on literature values for stable therapeutic peptides.
   - **Connection to Goals**: These criteria ensure that only peptides with high therapeutic potential are pursued, aligning with the project's clinical value objective.

8. **Risk Management**
   - **Explanation**: Potential risks include sample degradation and unexpected aggregation patterns. These will be mitigated through careful sample handling and pre-experiment stability assessments. Contingency plans include alternative experimental setups and data backup protocols. Monitoring will involve regular checks on sample integrity and aggregation state.
   - **Connection to Goals**: Proactive risk management ensures smooth project progression within the 3-month timeline, supporting the agenda's constraints.

9. **Simplification and Clarity**
   - **Explanation**: The rationale for each parameter is succinctly tied to the specific needs of the peptides, ensuring clarity and focus on project goals.

By incorporating these enhancements, the experimental setup is now more strategically aligned with the agenda, ensuring that each aspect of the experiment contributes to the overall success of the drug discovery project.

[end summary 4]

[begin summary 5]

To address the Scientific Critic's feedback and further align with the agenda, I will provide more detailed explanations on the integration with machine learning, specify sample preparation steps, and strengthen the justification for experimental choices.

### Revised ExperimentThermalShift Implementation

```plaintext
ExperimentThermalShift[
    myPeptideSamples,
    Instrument -> Model[Instrument, MultimodeSpectrophotometer, "Uncle"],
    NumberOfCycles -> 2,
    TemperatureRampRate -> (0.1 * Celsius / Second),
    TemperatureRamping -> Linear,
    MinTemperature -> (25 * Celsius),
    MaxTemperature -> (95 * Celsius),
    DynamicLightScattering -> True,
    AliquotAmount -> 10 * Microliter,
    AliquotContainer -> Model[Container, Plate, "96-well 2mL Deep Well Plate"]
]
```

### Justification and Improvements

1. **Integration with Machine Learning**:
   - **Data Utilization**: The thermal stability and DLS data will be used to train neural networks and regression models by serving as both training and validation datasets. These models will incorporate features such as melting temperature (Tm) and aggregation onset temperature to predict peptide stability and aggregation tendencies. This integration will allow us to refine peptide sequences for improved clinical efficacy, with expected improvements in prediction accuracy and model robustness.

2. **Sample Preparation Details**:
   - **Buffer and Dilution**: Samples will be diluted using a phosphate-buffered saline (PBS) solution at pH 7.4 to maintain physiological relevance. This ensures consistent sample preparation and optimal conditions for thermal shift analysis.

3. **Justification for Temperature Range**:
   - **Supporting Data**: The temperature range of 25°C to 95°C is supported by studies such as Smith et al. (2022) and preliminary stability tests conducted in our lab, indicating that our peptides maintain structural integrity within this range. This ensures that we capture relevant thermal transitions without risking peptide degradation.

4. **Dynamic Light Scattering (DLS) Measurements**:
   - **Quantitative Integration**: Aggregation onset temperatures identified through DLS will be used as features in our machine learning models. This quantitative integration will guide the selection of training data and inform peptide design modifications, enhancing the predictive power of our models.

5. **Risk Management and Contingency Plans**:
   - **Specific Plans**: In the event of unexpected peptide degradation, alternative formulations such as cyclized peptides or those with stabilizing modifications will be explored. Adjustments to the thermal profile, such as modifying the ramp rate or temperature range, will be based on initial results to optimize stability.

6. **Explicit Connection to Agenda Goals**:
   - **Clinical Value and Impact**: Ensuring thermal stability directly contributes to the clinical efficacy of peptide therapeutics by confirming their stability under physiological conditions. The integration of empirical data into machine learning models enhances predictive power, aligning with the agenda's goal of impactful application of machine learning in drug discovery.

7. **Simplification and Clarity**:
   - **Summary of Key Points**: The integration of thermal stability and DLS data into machine learning models will occur through iterative feedback loops, allowing for continuous refinement of predictions and experimental strategies. This process ensures that our models remain accurate and relevant to clinical applications.

By incorporating these improvements, the ExperimentThermalShift protocol is now more aligned with the project's goals, ensuring that it contributes effectively to both the scientific impact and clinical value of our research. This approach demonstrates a clear connection between experimental decisions and the overarching objectives of the project.

[end summary 5]

Here is the agenda for the meeting:

Please read the summaries of multiple separate meetings about the same agenda. Based on the summaries, provide a single answer that merges the best components of each individual answer. Please use the same format as the individual answers. Additionally, please explain what components of your answer came from each individual answer and why you chose to include them in your answer.

As a reference, here is the agenda from those meetings, which must be addressed here as well:

You are working on a research project to use machine learning for drug discovery. Your goals are the following: (1) the project must have high clinical value, meaning the research contributes to helping patients, (2) the project must include a scientifically impactful application of machine learning to drug discovery, and (3) the project must use Emerald Cloud Labs (ECL) for all experimental validation with a 3-month limit on experiments. Your team previously decided to pursue peptide-based therapeutics. You previously designed experimental protocols for synthesis and validation of peptides that are designed to bind to a given protein drug target. Now you must implement one of these protocols using ECL. Please implement the ExperimentThermalShift protocol using the appropriate commands and syntax from ECL, writing out the experiment in the form ExperimentThermalShift[inputs] where you fill in "inputs" with the appropriate inputs. Please refer to the ECL documentation that is provided as context with example function calls and the full set of options for the experiment. In general, leave most options at their default values and therefore do not include them in your experiment function call. Only specify options that must be changed from their default value for your specific experiment. For every option that you specify, please state the default value, the different value you chose, and why you chose that value instead of the default.



Medicinal Chemist, please provide your response to the agenda.

## Medicinal Chemist

### Revised ExperimentThermalShift Implementation

```plaintext
ExperimentThermalShift[
    myPeptideSamples,
    Instrument -> Model[Instrument, MultimodeSpectrophotometer, "Uncle"],
    NumberOfCycles -> 2,
    TemperatureRampRate -> (0.1 * Celsius / Second),
    TemperatureRamping -> Linear,
    MinTemperature -> (25 * Celsius),
    MaxTemperature -> (95 * Celsius),
    DynamicLightScattering -> True,
    AliquotAmount -> 10 * Microliter,
    AliquotContainer -> Model[Container, Plate, "96-well 2mL Deep Well Plate"]
]
```

### Justification and Integration with Agenda Goals

1. **Instrument Choice**
   - **Default Value**: Automatic (set to Model[Instrument, MultimodeSpectrophotometer, "Uncle"] for protein samples)
   - **Chosen Value**: Model[Instrument, MultimodeSpectrophotometer, "Uncle"]
   - **Rationale**: The Uncle spectrophotometer is selected for its dual capability to measure fluorescence and static light scattering, providing comprehensive data on both thermal stability and aggregation. This is crucial for understanding peptide behavior under thermal stress. This choice is consistently supported across summaries 1, 2, 3, and 4.

2. **Number of Cycles**
   - **Default Value**: Automatic (set to 0.5 cycles for ViiA 7)
   - **Chosen Value**: 2
   - **Rationale**: Two cycles ensure data reproducibility, allowing us to confirm consistent thermal stability across multiple runs. This choice is emphasized in summaries 2, 3, and 4 for its importance in capturing reversible structural changes.

3. **Temperature Ramp Rate**
   - **Default Value**: Automatic (set to 1 Celsius/Minute for Linear)
   - **Chosen Value**: 0.1 Celsius/Second
   - **Rationale**: A ramp rate of 0.1 Celsius/Second is chosen to capture subtle changes in peptide stability, balancing experimental duration with resolution. This choice is highlighted in summaries 2, 3, 4, and 5 for its ability to provide detailed thermal profiles.

4. **Temperature Range**
   - **Default Values**: 15 degrees Celsius and 95 degrees Celsius
   - **Chosen Values**: 25 degrees Celsius and 95 degrees Celsius
   - **Rationale**: Starting at 25°C minimizes environmental noise, while 95°C captures the full range of thermal transitions for proteins. This choice is supported in summaries 2, 3, and 5 for its relevance to capturing key thermal transitions.

5. **Dynamic Light Scattering (DLS)**
   - **Default Value**: False
   - **Chosen Value**: True
   - **Rationale**: DLS provides insights into peptide aggregation, informing design strategies to minimize undesirable aggregation. This choice is consistently emphasized in summaries 2, 3, 4, and 5 for its role in understanding peptide solubility and stability.

6. **Aliquot Amount and Container**
   - **Default Value**: Not specified
   - **Chosen Value**: 10 Microliters, Model[Container, Plate, "96-well 2mL Deep Well Plate"]
   - **Rationale**: These settings ensure consistent sample preparation and optimal conditions for thermal shift analysis. This choice is specifically detailed in summary 5, aligning with the need for precise sample handling.

7. **Integration with Machine Learning**
   - **Explanation**: Thermal shift data, including Tm and aggregation behavior, will refine machine learning models by providing real-world validation of predicted stability and binding affinity. We will employ neural networks to predict peptide stability and support vector machines for binding affinity predictions. Feedback loops will occur bi-weekly, allowing for timely model updates and design adjustments. This integration is a key component in summaries 1, 2, 3, and 5, ensuring effective use of machine learning.

8. **Risk Management and Contingency Plans**
   - **Explanation**: Potential risks include sample degradation and unexpected aggregation patterns. These will be mitigated through careful sample handling and pre-experiment stability assessments. Contingency plans include alternative experimental setups and data backup protocols. Monitoring will involve regular checks on sample integrity and aggregation state. This proactive approach is highlighted in summaries 4 and 5.

9. **Connection to Agenda Goals**
   - **Clinical Value and Impact**: Ensuring thermal stability directly contributes to the clinical efficacy of peptide therapeutics by confirming their stability under physiological conditions. The integration of empirical data into machine learning models enhances predictive power, aligning with the agenda's goal of impactful application of machine learning in drug discovery. This alignment is emphasized across all summaries.

By incorporating these elements, the ExperimentThermalShift protocol is strategically aligned with the project's goals, ensuring that it contributes effectively to both the scientific impact and clinical value of our research. This approach demonstrates a clear connection between experimental decisions and the overarching objectives of the project.

